src/test/groovy/nebula/plugin/extraconfigurations/OptionalBasePluginIntegrationTest.groovy

```diff
@@ -125,89 +125,7 @@ publishing {
         runTasksSuccessfully('publish')
 
         then:
-        assertOptionalDependencyInGeneratedPom(repoUrl, 'org.apache.commons', 'commons-lang3', '3.3.2', 'compile')
```

Comment:
`maven` plugin is being deprecated so I removed its usage.

Reply:
There's a couple items to look at in the README, too:

> provided - We recommend the built in compileOnly configuration for Gradle releases that support it. Incompatible with Gradle versions 3.4 and later and will warn it does nothing

and 

> The following publishing plugins support the correct handling when generating the relevant metadata:
> 
> Maven
> Maven Publishing
> Ivy Publishing

Can you update the README to remove `maven` support?

And since `provided` stopped working with Gradle 3.4 & later, I think we can remove info about that from the readme as well